playbooks/robusta_playbooks/alerts_integration.py

```diff
@@ -238,11 +250,22 @@ def logs_enricher(event: PodEvent, params: LogEnricherParams):
                 ],
             )
         return
+
     log_data = pod.get_logs()
-    if log_data:
-        event.add_enrichment(
-            [FileBlock(f"{pod.metadata.name}.log", log_data.encode())],
-        )
+    if not log_data:
+        return
+
+    if params.regex_replacer_pattern:
+        regex_replacement_style = RegexReplacementStyle[params.regex_replacement_style]
+        logging.info('Sanitizing log data with the provided regex pattern')
```

Comment:
I would add the filtering regex

Reply:
My original code had this too, but the filtering regex may contain sensitive information.